optimus_prime/Source.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+
+
+bool is_prime(int number) {
+	if (number < 2) {
+		return false;
+	}
+
+	for (int i = 2; i < number; ++i) {
```

Comment:
Do we really need to go all the way from 2 to number?
If number == 30, do we need to check if 30 % 20 == 0? do we need to check 30 % 18? 30 % 10? 
Figure out a way to decrease the complexity of this method.
Feel free to talk this with me 1 on 1 in the private chat room if you're not sure what to do.